lib/generators/component/templates/view.html.erb.erb

```diff
@@ -1,3 +1,3 @@
 <div class="<%= component_class_name %>">
-  <%= component_name %>
+  <%%= t "<%= module_name.underscore %>.component_name" %>
```

Comment:
Locale is optional, maybe a simple if / else statement should be ok to ensure we not calling a key missing.

Reply:
Or maybe specify a default? http://guides.rubyonrails.org/i18n.html#defaults